app/src/main/java/com/keylesspalace/tusky/components/account/AccountViewModel.kt

```diff
@@ -65,6 +67,8 @@ class AccountViewModel @Inject constructor(
                             accountData.postValue(Success(account))
                             isDataLoading = false
                             isRefreshing.postValue(false)
+
+                            isFromOwnDomain = getDomain(account.url) == accountManager.activeAccount?.domain
```

Comment:
There are race conditions which mean that `accountManager.activeAccount` can change over the life of the activity (or fragment).

Technically it's not a problem here, but I'm trying to remove all occurrences of this pattern in the code.

You can put

```kotlin
val activeAccount = accountManager.activeAccount!!
```

at the start of the class, and then use just `activeAccount.domain` here.

Reply:
I've addressed this and made sure to use the field on `setAccountInfo()` too.